go/vt/vttablet/endtoend/config_test.go

```diff
@@ -176,11 +177,24 @@ func TestConsolidatorReplicasOnly(t *testing.T) {
 }
 
 func TestQueryPlanCache(t *testing.T) {
+	if cache.DefaultCacheSize.Bytes() != 0 {
```

Comment:
shouldn't we test both?

Reply:
This is the case where testing both is not feasible, as this is an end2end test with a global Vitess instance that cannot be re-configured without tests.